interface/patient_file/deleter.php

```diff
@@ -209,9 +209,9 @@ function delete_document($document)
 
 <script>
 function submit_form() {
-    top.restoreSession();
```

Comment:
hi @tpantelis , Was this line breaking anything. We generally have to make a call to restoreSession from the top frame whenever submit or go to another script (this ensures that different sessions in same browser do not get mixed up: 
https://www.open-emr.org/wiki/index.php/OpenEMR_System_Architecture#PHP_Sessions_and_Browser_Windows
(the reason OpenEMR does this is to allow a user to open different sessions/logins when working on different patients; the goal at some point hopefully in near future is to rid OpenEMR of this requirement: https://github.com/openemr/openemr/issues/2556)

Reply:
hmm... not sure how/why that was removed. I put it back.